### PR TITLE
chore(coverage): Update instrumenter

### PR DESCRIPTION
**Pull request by TheSharpieOne**
Replace isparta with istambul since isparta is no longer maintained
See: https://github.com/douglasduteil/isparta
Istambul now support es2015/6.
(Also added cross-env to allow it to work on windows)

This PR is not a big deal, just some maintenance, though I did notice the coverage output was _slightly_ more detailed with the babel istambul plugin this PR adds than the isparta loader it replaces.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -46,6 +46,7 @@
   "dependencies": {
     "babel-cli": "^6.11.4",
     "classnames": "^2.2.3",
+    "cross-env": "^2.0.0",
```
Comment (eddywashere) [inline package.json]: I know it's weird, but I was hoping to restrict `dependencies` to the core reactstrap library dependencies. I must have accidently added `babel-cli` to it.

**Comment by eddywashere** [comment]
I think I should restrict tests to latest node version to avoid the double coverage reports.

**Comment by TheSharpieOne** [comment]
Yeah, not too sure why it is not about to determine the changes to the coverage either and the branches are "**"... strange.

File: package.json (hunk 1 above)
Comment (TheSharpieOne) [inline package.json]: will do

File: package.json [hunk 2]
```diff
@@ -56,8 +55,10 @@
     "react-dom": "^0.14.0 || ^15.0.0"
   },
   "devDependencies": {
+    "babel-cli": "^6.11.4",
```
Comment (eddywashere) [inline package.json]: 🙌  thanks!

**Pull request merged**